ElectronClient/gui/ConfigScreen.jsx

```diff
@@ -307,6 +331,15 @@ class ConfigScreenComponent extends React.Component {
 						style={selectStyle}
 						onChange={event => {
 							updateSettingValue(key, event.target.value);
+							// if Auto Detect option is ON theme should change according to preferred theme options
+							if (settings['themeAutoDetect']) {
+								// preferred dark theme should only work for dark system theme
+								if (key == 'preferredDarkTheme' && this.state.shouldUseDarkColors) {
```

Comment:
`shouldUseDarkColours` could be 'useDarkTheme'

Reply:
I keep using `shouldUseDarkColours` name because as @laurent22 mentioned in earlier comment 

> We have a nicely named function `shouldUseDarkColors` and you create a wrapper and rename it to `systemTheme`, which is meaningless. Is it a boolean, is it a string? What are the possible values? Please just keep the name `shouldUseDarkColors`

So after that I didn't thought of changing the name other than `shouldUseDarkColors` but now as you are suggesting `useDarkTheme` sounds good to me. I'll change state name `shouldUseDarkColors` to `useDarkTheme`

Reply:
No, sorry I didn't see that comment, and I did not mean to confuse the issue.  Please don't make the change. Follow Laurent's advice, as you had done..

Reply:
Yea Sure 😃